source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -390,15 +391,15 @@ def filter_func(self, journey):
         return max_duration > journey.duration
 
 
-def get_min_connections(journeys):
+def get_best_pt_journey_connections(journeys, request):
     """
-    Returns min connection count among journeys
+    Returns the nb of connection of the best pt_journey
     Returns None if journeys empty
     """
     if not journeys:
         return None
-
-    return portable_min((get_nb_connections(j) for j in journeys if not to_be_deleted(j)), default=0)
+    best = get_ASAP_journey((j for j in journeys if 'non_pt' not in j.tags), request)
```

Comment:
Why not among all journeys?
If direct walking is the best, we should filter from that, no?

Reply:
it'll be problematic in distributed, because a direct path will always appear in the response. with direct path by car/bike/taxi, that may shoot a lot of interesting pt journey

Reply:
OK.
:bulb: Idea if this filter becomes too permissive (later): we may consider the direct walk journey (if there is some) as a possible reference.

Reply:
This is the kind of filter that will have to evolve when we will work to improve multimodal journeys